docs/api.rst

```diff
@@ -5381,7 +5381,7 @@ Button
     :members:
     :inherited-members:
 
-.. autofunction:: disnake.ui.button
+.. autofunction:: disnake.ui.button(cls=disnake.ui.Button, *, style=ButtonStyle.secondary, label=None, disabled=False, custom_id=None, url=None, emoji=None, row=None)
```

Comment:
would you please put this in the same order as disnake.ui.select?
At the very least custom_id should be at the beginning and likely with `...` as the default as well. While it is not entirely accurate it doesn't need to be.

Reply:
Verdict reached on [discord](https://canary.discord.com/channels/808030843078836254/913779868985090089/1000157708055494656) was that we won't make this change right now; changed the default though to `...` as requested.